dev/bench/gitlab.sh

```diff
@@ -40,18 +40,19 @@ echo $PWD
 #check_variable "JOB_NAME"
 #check_variable "JENKINS_URL"
 check_variable "CI_JOB_URL"
-check_variable "coq_pr_number"
-check_variable "coq_pr_comment_id"
-check_variable "new_ocaml_switch"
-check_variable "new_coq_repository"
-check_variable "new_coq_opam_archive_git_uri"
-check_variable "new_coq_opam_archive_git_branch"
-check_variable "old_ocaml_switch"
-check_variable "old_coq_repository"
-check_variable "old_coq_opam_archive_git_uri"
-check_variable "old_coq_opam_archive_git_branch"
-check_variable "num_of_iterations"
-check_variable "coq_opam_packages"
+
+coq_pr_number=""
+coq_pr_comment_id=""
+new_ocaml_switch="ocaml-base-compiler.4.07.1"
+old_ocaml_switch="ocaml-base-compiler.4.07.1"
+new_coq_repository="https://gitlab.com/coq/coq.git"
+old_coq_repository="https://gitlab.com/coq/coq.git"
+new_coq_opam_archive_git_uri="https://github.com/coq/opam-coq-archive.git"
+old_coq_opam_archive_git_uri="https://github.com/coq/opam-coq-archive.git"
+new_coq_opam_archive_git_branch="master"
+old_coq_opam_archive_git_branch="master"
+num_of_iterations=1
+coq_opam_packages="coq-performance-tests-lite coq-engine-bench-lite coq-hott coq-bignums coq-mathcomp-ssreflect coq-mathcomp-fingroup coq-mathcomp-algebra coq-mathcomp-solvable coq-mathcomp-field coq-mathcomp-character coq-mathcomp-odd-order coq-math-classes coq-corn coq-flocq coq-compcert coq-geocoq coq-color coq-coqprime coq-coqutil coq-bedrock2 coq-rewriter coq-fiat-core coq-fiat-parsers coq-fiat-crypto coq-unimath coq-sf-plf coq-coquelicot coq-lambda-rust coq-verdi coq-verdi-raft coq-fourcolor coq-rewriter-perf-SuperFast"
```

Comment:
Maybe using `?=` instead of `=` to be able to override them from the UI?

Reply:
That's not shell syntax, but I can use the shell syntax.

Reply:
Changed